docs/changes.rst

```diff
@@ -4,6 +4,9 @@ Change log
 Pending
 -------
 
+* Added support for Python 3.13 to the CI matrix.
+* Removed support for Python 3.8 as it is end of life.
```

Comment:
Another suggestion. The first line is optional, I feel more strongly about the second line.

```suggestion
* Added Python 3.13 to the CI matrix.
* Removed support for Python 3.8 as it has reached end of life.
```

Reply:
Added.